### PR TITLE
Adds github actions to build python 2.7 and 3.7

### PR DESCRIPTION
**Pull request by plasmax**
Provides a github actions build system for python 2.7 and 3.7 wheels, containing all dependencies (except python itself).

Based off alembic#335 for Python **2.7** Wheel.
Continuation of framestore/alembic#1 pull request for Python **3.7** Wheel.

Uses Docker for both builds, with the aswf/ci-usd:2021.6 image and Imath v3.0.4 for the Python 3.7 build.

Runs tests found in PyAlembic/Tests after the wheel builds are complete.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -58,19 +103,17 @@ def build_extension(self, ext):
...
+        subprocess.check_call(
+            [cmake_command, ext.sourcedir] + cmake_args, cwd=self.build_temp, env=env)
+        subprocess.check_call(
+            [cmake_command, '--build', '.'] + build_args, cwd=self.build_temp)
+
+
 setup(
     name='alembic',
-    version='1.7.16',
+    version='1.8.2',
```
Comment (manneohrstrom) [inline setup.py]: please add a comment to highlight that this needs to be manually updated to match the currrent abc version!

**Review by manneohrstrom — APPROVED** [review]
hey! Looks great! Sorry for taking time to get this reviewed.

The thing that is missing that we should add is some docs/readme that explains 

- what it is
- how to run it and get wheels built for a given version of alembic

Thanks for all your work Max!

File: setup.py (hunk 1 above)
Comment (plasmax) [reply setup.py]: https://github.com/framestore/alembic/blob/919f4f4cb914b5a9936e6d6f315f618c16e56ea3/setup.py#L114-L116

Added comment about versioning.

**Comment by plasmax** [comment]
> 
> 
> hey! Looks great! Sorry for taking time to get this reviewed.
> 
> The thing that is missing that we should add is some docs/readme that explains
> 
>     * what it is
> 
>     * how to run it and get wheels built for a given version of alembic
> 
> 
> Thanks for all your work Max!

Thanks Manne! Documentation for the python manylinux wheels build process added [here](https://github.com/framestore/alembic/blob/919f4f4cb914b5a9936e6d6f315f618c16e56ea3/WHEELS.md)

**Pull request merged**